src/sodaspark/scan.py

```diff
@@ -264,19 +265,26 @@ def create_scan(
     variables: variables to be substituted in scan yml
     soda_server_client : Optional[SodaServerClient] (default : None)
         A soda server client.
+    time: timestamp date in ISO8601 format (defaul: None)
```

Comment:
Could you write the same as in the function below:

```
<parameter name> : <parameter type> (default: <default value)
    <parameter description
```

And add to the description what happens if `None` -> use `now...`